model/DataObject.php

```diff
@@ -1771,6 +1771,9 @@ public function getManyManyComponents($componentName, $filter = null, $sort = nu
 			$componentClass, $table, $componentField, $parentField,
 			$this->manyManyExtraFieldsForComponent($componentName)
 		);
+		
+		$this->extend('updateManyManyComponents', $result);
```

Comment:
This is a good idea, but I would like to see it after the `setDataModel` call, ideally.